gui/gui/src/TGFileBrowser.cxx

```diff
@@ -1317,7 +1331,7 @@ void TGFileBrowser::DoubleClicked(TGListTreeItem *item, Int_t /*btn*/)
             // than a canvas already embedded in one of the browser's tab
             obj->DrawClone();
          }
-         else if (fBrowser && !obj->InheritsFrom("TFormula"))
+         else if (fBrowser && !obj->InheritsFrom("TFormula") && !is_rntuple) //!is_rntuple prevents the browse method from being called here.
```

Comment:
Yes definitely, I checked once again to make sure. Removing this would make TBrowser display stuff like `NTPLP0_0;1`

Reply:
> Removing this would make TBrowser display stuff like NTPLP0_0;1

Why is that a problem (that the browser display the cycles of the RNtuples?)

Reply:
Without `&& !is_rntuple`, it would be displayed like this:
<img width="232" alt="Screenshot 2019-08-21 at 08 39 03" src="https://user-images.githubusercontent.com/45257539/63408506-7fc92700-c3ef-11e9-9e28-dd883c26f8b5.png">

Reply:
Ah, thanks!  Perhaps clarify in the comment that it prevents RNTuple's internal key names from being displayed.

Reply:
I will do so!